scss/grid.scss

```diff
@@ -0,0 +1,89 @@
+.flex-body { 
+  grid-template-areas: 
+    "header"
+    "main"
+    "footer";
+
+  .train-header {
+    align-items: center;
+    display: grid;
+    grid-area: header;
+    grid-template-areas: 
+      "logo . dev";
```

Comment:
Do this inline `grid-template-areas: "logo . dev";`
You should use flexbox instead tho, I dont like that `.`